yelp_package/dockerfiles/bionic/Dockerfile

```diff
@@ -18,7 +18,7 @@ FROM ${DOCKER_REGISTRY}ubuntu:bionic
 ARG PIP_INDEX_URL=https://pypi.yelpcorp.com/simple
 ENV PIP_INDEX_URL=$PIP_INDEX_URL
 
-RUN sed -i 's/archive.ubuntu.com/us-east1.gce.archive.ubuntu.com/g' /etc/apt/sources.list
+# RUN sed -i 's/archive.ubuntu.com/us-east1.gce.archive.ubuntu.com/g' /etc/apt/sources.list
```

Comment:
nit: should we just delete this line and the commented mesosphere stuff in all the dockerfiles?